app/commands/dailyRecGraph.js

```diff
@@ -0,0 +1,11 @@
+const prepareAnswer = require('../utils/prepareAnswer');
```

Comment:
Let's follow the convention and keep the name of this file as `dailyRecoveredGraph.js`

Reply:
okay no problem changing it 👍